### ISSUE TITLE
Mini-cart hide the content of page while she  in closed state

### ISSUE THREAD

**Issue by navarroido** [elementor/elementor#16305]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

When the cart is closed, it still appears on the page, probably in full transparency mode. So when you move the mouse over an area that is supposed to be open when it appears, you are unable to access the actual content on the page because it is in the upper layer, probably a high but hidden Z-index.

Video:
https://www.loom.com/share/c94cf955d1fe44fbb3d5360a501ba95b

## Steps to reproduce

Video:
https://www.loom.com/share/c94cf955d1fe44fbb3d5360a501ba95b

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.35
	PHP Version: 7.3.30
	PHP Max Input Vars: 10000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.3
	Site URL: https://wordpress-themes.co.il/demo-plugins
	Home URL: https://wordpress-themes.co.il/demo-plugins
	WP Multisite: Yes
	Max Upload Size: 5 MB
	Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: he-IL
	Timezone: Asia/Jerusalem
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: he_IL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36

== Network Plugins ==
	Buy Now Button for WooCommerce
		Version: 1.0.0
		Author: Ido Navarro

	Elementor
		Version: 3.4.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.4.1
		Author: Elementor.com

	Flying Product to Mini Cart for WooCommerce Pro
		Version: 1.0.0
		Author: ido navarro

	WooCommerce
		Version: 5.6.0
		Author: Automattic


== אלמנטור - Compatibility Tag ==
	


== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	footer : 1
		heading : 1
		icon-list : 1
		image : 1
		nav-menu : 1
		social-icons : 1
	header : 1
		nav-menu : 1
		theme-site-logo : 1
		woocommerce-menu-cart : 1
	popup : 1
		form : 1
		heading : 1
		icon : 2
		spacer : 4
		text-editor : 1
	product : 1
		heading : 1
		woocommerce-breadcrumb : 1
		woocommerce-product-add-to-cart : 1
		woocommerce-product-images : 1
		woocommerce-product-meta : 1
		woocommerce-product-price : 1
		woocommerce-product-related : 1
		woocommerce-product-short-description : 1
		woocommerce-product-title : 1
	single : 1
		button : 1
		heading : 2
		image : 1
		spacer : 7
	wp-page : 7
		animated-headline : 4
		button : 1
		form : 1
		heading : 11
		icon : 4
		icon-list : 1
		image : 2
		lottie : 1
		menu-anchor : 3
		spacer : 31
		text-editor : 11
		toggle : 1
		wc-elements : 2
		woocommerce-products : 1
	product-archive : 1
		heading : 2
		lottie : 1
		spacer : 1
		theme-archive-title : 1
		woocommerce-breadcrumb : 1
		woocommerce-products : 1


== Log ==
	
Log: showing 14 of 142021-09-09 19:50:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.2.3',
  'to' => '3.4.3',
)]
2021-09-09 19:50:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.4.0',
)]
2021-09-09 19:50:38 [info] elementor::elementor_updater Started 
2021-09-09 19:50:38 [info] אלמנטור/Upgrades - _on_each_version Start  
2021-09-09 19:50:38 [info] אלמנטור/Upgrades - _on_each_version Finished 
2021-09-09 19:50:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.2.3',
  'to' => '3.4.3',
)]
2021-09-09 19:50:39 [info] elementor-pro::elementor_pro_updater Started 
2021-09-09 19:50:39 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-09-09 19:50:39 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-09-09 19:50:39 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-09-09 19:50:39 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2021-09-09 19:50:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.4.0',
)]
2021-09-09 19:50:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.2.3',
  'to' => '3.4.3',
)]
2021-09-09 19:50:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.4.0',
)]

PHP: showing 1 of 1PHP: 2021-09-09 19:58:33 [notice X 1][/home/wordpressth/public_html/wp-content/plugins/elementor/core/app/modules/import-export/module.php::104] filter_var(): explicit use of FILTER_FLAG_SCHEME_REQUIRED and FILTER_FLAG_HOST_REQUIRED is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by avivu** [comment]
Hello @navarroido,

We tried to reproduce this issue, but with no luck.
Can you provide more details with system info?
Also, What browser are you using?

Thank you!

**Comment by navarroido** [comment]
> Hello @navarroido,
> 
> We tried to reproduce this issue, but with no luck.
> Can you provide more details with system info?
> Also, What browser are you using?
> 
> Thank you!

All info attach to post on "Environment"
You can also look  on the live website under the URL:
https://wordpress-themes.co.il/demo-plugins

**Comment by avivu** [comment]
Thank you, @navarroido,
We will take a look and investigate the issue.

**Comment by avivu** [comment]
This issue has been resolved in Elementor Pro v3.4.2

Feel free to update

Thanks!
